godb/tasks/docker.yml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
This could be a candidate for a separate role in future? But fine for now.

Reply:
agreed!